Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -40,6 +40,12 @@ SECTORDER_FLAGS = $(SECTORDER_FLAGS_$(CONFIGURATION));
 SECTORDER_FLAGS_Production[sdk=iphoneos*] = -Wl,-order_file,$(SDKROOT)/AppleInternal/OrderFiles/JavaScriptCore.order;
 SECTORDER_FLAGS_Production[sdk=macosx*] = -Wl,-order_file,JavaScriptCore.order;
 
+PROFILE_DATA_PATH = $(PROFILE_DATA_PATH_PRODUCTION_$(DEPLOYMENT_LOCATION));
+PROFILE_DATA_PATH_PRODUCTION_YES = $(SDK_DIR)/usr/local/include/WebKitAdditions/Profiling/JavaScriptCore.profdata.compressed;
```

Comment:
Not a regression from status quo, but makes me think that there are many cases where we fail to find the data files. Without getting into details, this computation is much more naive than those for search or product paths.

Reply:
Agreed, we can certainly add complexity here as needed.